.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @up42-eva @javidq @andher1802 @jmigueldelgado @seedlit
+* @up42-eva @javidq @andher1802 @jmigueldelgado
```

Comment:
Let's rather change this to the github team so @abossart could handle this in the future